vpr/src/pack/cluster.cpp

```diff
@@ -153,7 +155,8 @@ static bool is_atom_blk_in_pb(const AtomBlockId blk_id, const t_pb* pb);
 static void add_molecule_to_pb_stats_candidates(t_pack_molecule* molecule,
                                                 std::map<AtomBlockId, float>& gain,
                                                 t_pb* pb,
-                                                int max_queue_size);
+                                                int max_queue_size,
+                                                AttractionInfo& attraction_groups);
```

Comment:
check if these can be passed as const ref, make the ones that are just reading const